user_interaction/static/themes/dark-theme.css

```
@@ -65,3 +65,7 @@ a:hover {
     color: white;
     background-color: red;
 }
+
+.bg-light {
```

This results in any bg-light to turn red and we can not guarantee that that is the case. It makes more sense to treat bg-light in a dark color scheme as a darker instance than bg-dark making it pop more with the light text.
Or, if you specifically want to target the alert require a combination of both bg-light and border-danger for this to trigger:
```suggestion
.bg-light.border-danger {
```